week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -11,13 +11,13 @@ We made a start on this for you here:
 let bottle = {
   volume: 0,
   fill: function() {
-    // calling this function should make you bottles volume = 100;
+    this.volume = 100; // calling this function should make you bottles volume = 100;
   },
   drink: function() {
-    // calling this function should decrease your bottles volume by 10;
+    this.volume -= 10;// calling this function should decrease your bottles volume by 10;
   },
   empty: function() {

```

Comment:
What would be if you use bottle.volume ? what is the difference with this.volume?

Reply:
YEs, map could do as well